streamly.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      2.2
 name:               streamly
-version:            0.7.3
+version:            0.7.3.2
```

Comment:
You should probably branch this from the latest 0.7.3.*

Reply:
Once #2421 merges, I'll be able to do this won't I?

Reply:
Resolved now. I've changed the base branch.